index.html

```diff
@@ -15,27 +15,41 @@
         </a>
 
         <nav class="menu">
-            <svg  width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
-                <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z" fill="white"/>
-            </svg>
+
+            <div class="menu-desktop">
+                <a class="element" href="">Portfolio</a>
+                <a class="element" href="">About</a>
+                <a class="element" href="">Contact</a>
+                <img src="Images/Main/Msg-Icon.png" alt="">
+            </div>
+
+            <div class="hamb-icon">
+                <svg  width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z" fill="white"/>
+                </svg>
+            </div>
```

Comment:
You did well with your design, please kindly reduce the margin so as to allow your portfolio desktop navigation to be similar to the design for a better UI  and also reduce the background image width as it appears dragged over the section😄 

Figma design :
![image](https://github.com/ulises2607/Portfolio/assets/20537712/50adfde9-3e02-40d7-be0f-36d4689c7c3b)

Your design :
![image](https://github.com/ulises2607/Portfolio/assets/20537712/4f15cb04-7460-4d75-b797-74f4636a42e4)